metaflow/client/core.py

```diff
@@ -290,23 +291,40 @@ def __init__(
         if pathspec:
             ids = pathspec.split("/")
 
+            if self._NAME == "flow" and len(ids) != 1:
+                raise MetaflowInvalidPathspec("Expects Flow('FlowName')")
+            elif self._NAME == "run" and len(ids) != 2:
+                raise MetaflowInvalidPathspec("Expects Run('FlowName/RunID')")
+            elif self._NAME == "step" and len(ids) != 3:
+                raise MetaflowInvalidPathspec(
+                    " Expects Step('FlowName/RunID/StepName')"
```

Comment:
nit: addl space in front of `Expects` -- same in the next 2 instances.

Reply:
Fixed, thanks!